### DISCUSSION TITLE
How do YOU sort  the 99-unsorted.sh file?

### DISCUSSION THREAD

**Discussion by akim-13** [CyberShadow/aconfmgr#150] [Q&A]
I've just started using this wonderful program and it would've been very helpful if somebody provided their sorted shell files, so that I could get an idea of how it can be done in practice.

Yes, I've read README.
>Once `aconfmgr save` finishes, you should review the contents of `99-unsorted.sh`, and sort it into one or more new files (e.g.: `10-base.sh`, `20-drivers.sh`, `30-gui.sh`, `50-misc.sh` ...).

**Comment by CyberShadow** [comment]
You can find some aconfmgr configurations by searching GitHub, for example:

- https://github.com/compactcode/arch
- https://github.com/infokiller/config-public/tree/master/install/aconfmgr
- https://github.com/jfly/dotfiles/tree/master/aconfmgr-config
- https://github.com/OJFord/dotfiles/tree/master/aconfmgr
- https://github.com/mikeroll/dotfiles/tree/arch/.config/aconfmgr
- https://github.com/sakhnik/alarmpi3-config
- https://github.com/zoickx/dotfiles/tree/master/aconfmgr
